commander/src/commander-plugin.c

```diff
@@ -345,7 +345,16 @@ store_populate_menu_items (GtkListStore  *store,
       } else {
         gchar *tmp;
         gchar *tooltip;
-        gchar *label = g_markup_printf_escaped ("<big>%s</big>", item_label);
+        gchar *basename = g_path_get_basename (item_label);
+        gchar *dirname = g_path_get_dirname (item_label);
+        gchar *dirname_basename = g_path_get_basename(dirname);
+        gchar *label;
+
+        if (g_strcmp0(".", dirname_basename)  == 0) {
+          label = g_markup_printf_escaped ("<big>%s</big>", basename);
+        } else {
+          label = g_markup_printf_escaped ("<big>%s/%s</big>", dirname_basename, basename);
+        }
```

Comment:
This seems overly fragile to me, as menu item labels can contain pretty much anything, including `/`s.

Take for example *Edit → Insert Date → yyyy/mm/dd*: with this change, it appears as *mm/dd*.

Reply:
Yes, that may be a problem, and in that case, it's also possible to see all details in the smaller part of the menu item ( `Edit → Insert Date → yyyy/mm/dd`).